testController.js

```diff
@@ -1,8 +1,37 @@
 document.addEventListener('DOMContentLoaded', function() {
-    document.getElementById('home').addEventListener('click', function() {
-        console.log('vleze')
-        getWeather('sofia').then(function(data) {
-            console.log(data)
+        function getCity() {
+            return document.getElementById('city').value
+        }
+    document.getElementById('enterCity').addEventListener('click', function() {
+        let city = document.getElementById('city').value 
```

Comment:
It's better to get all references to elements on top and use them in the event listeners if these references dont change (for example when an element with specific id is removed/added dynamically). So if your element with 'city' id doesn't change it's reference, you can define it on top of the 'DOMContentLoaded' callback and use it here.